projects/i18n/switch/load-lang.ts

```diff
@@ -0,0 +1,19 @@
+import {TuiLanguage, TuiLanguageName} from '@taiga-ui/i18n/interfaces';
+import {from, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {TuiLangChunkLoader} from './options';
+
+export function tuiLoadLanguage(
+    language: TuiLanguageName,
+    loader: Exclude<TuiLangChunkLoader, null | undefined>,
```

Comment:
`undefined` is not in the type. Also there's a `NonNullable` utility type in TypeScript ;)

Reply:
fixed